Libraries/StyleSheet/ColorValueTypes.js

```diff
@@ -0,0 +1,25 @@
+/**
```

Comment:
Can you help me understand the benefit of this file?

Reply:
This file defines the Flow type `ColorValue` and the new Flow type `ProcessedColorValue`.   `ColorValue` used to be defined in StyleSheetTypes.js.   Since `ColorValue` is now based on the opaque `NativeColorValue` type from NativeColorValueTypes.<platform>.js it felt cleaner to do the "glueing" in this new file instead of in StyleSheetTypes.js.   I could drop it and move it all into StyleSheetTypes.js -- don't feel strongly one way or the other.

Reply:
It moves the defintion of `ColorValue` from StyleSheetTypes.js to here because `ColorValue` is now based on the opaque type `NativeColorValue` defined in NativeColorValueTypes.<platform>.js.   It just felt cleaner to do the "glueing" in this file instead of StyleSheetTypes.h.  I could drop it and move it to StyleSheetTypes.js -- don't feel strongly one way or the other.